src/members/locale/sv/LC_MESSAGES/django.po

```diff
@@ -31,11 +35,11 @@ msgstr "Personnummer på formen ÅÅÅÅMDD-NNNN."
 
 #: forms.py:75 forms.py:235
 msgid "A user with that username already exists."
-msgstr ""
+msgstr "En användare med det användrnamnet finns redan"
```

Comment:
felstavning

Reply:
Fortsättningsvis, även om det är för sent, måste vi ha username om vi kräver email?

Reply:
Ja för user modellen bygger på att varje konto har ett användarnamn och lösenord som fält i databasen. Vi skulle behöva göra om hela user modellen vilket blir en större förändring. Jag har skrivit upp det som en grej för Dkå att göra. Just med denna PR vill jag bara få in validering på telefonnummer och epost